src/templates_front/templator_front.org

```diff
@@ -703,23 +703,33 @@ struct trexio_back_end_s {
 *** C
 
    #+begin_src c :tangle prefix_front.h :exports none
-trexio_t* trexio_open(const char* file_name, const char mode, const back_end_t back_end);
+trexio_t* trexio_open(const char* file_name, const char mode, 
+                      const back_end_t back_end, trexio_exit_code* const rc_open);
    #+end_src
 
    #+begin_src c :tangle prefix_front.c
 trexio_t*
 trexio_open(const char* file_name, const char mode,
-	    const back_end_t back_end)
+	    const back_end_t back_end, trexio_exit_code* const rc_open)
 {
 
```

Comment:
We need to check here that `rc_open` is not `NULL`, because if it is `NULL` the code will crash at line 717.

We can let user who doesn't want to check the error code pass a `NULL` pointer, and update `rc_open` only if it is not `NULL`.

Reply:
Done.